### ISSUE TITLE
fashion-mnist url is error.

### ISSUE THREAD

**Issue by anlei1996** [tensorflow/tensorflow#21236]
TensorFlow website error url:https://www.tensorflow.org/tutorials/keras/basic_classification
it should downloads the fashion-mnist,but it really downloads the origin-mnist which is digital.
please modify this error.
I download the right datasets from GitHub:https://github.com/zalandoresearch/fashion-mnist

**Comment by av8ramit** [comment]
@anlei1996 

Are you referring to a link on the website or the following code snippet?
`
fashion_mnist = keras.datasets.fashion_mnist

(train_images, train_labels), (test_images, test_labels) = fashion_mnist.load_data()
`

**Comment by av8ramit** [comment]
Great! Closing this issue now.

**Comment by ypatz** [comment]
http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/t10k-labels-idx1-ubyte.gz
http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/t10k-images-idx3-ubyte.gz
Tensorflow 1.8 still wrong datasets.

**Comment by ypatz** [comment]
Yes, I use the same code.

**Comment by av8ramit** [comment]
@ypatz I'm unable to reproduce even with tensorflow==1.8.0. The dataset is fashion-mnist. What about it do you believe is wrong? Maybe I'm misunderstanding.

**Comment by av8ramit** [comment]
Closing the issue. Unable to reproduce on 1.8.0.
